core/atomic_c11.c

```diff
@@ -137,12 +137,12 @@ typedef uint64_t I8;
 #define TEMPLATE_ATOMIC_FETCH_OP_N(opname, op, n, prefixop) \
     I##n __atomic_fetch_##opname##_##n(volatile void *ptr, I##n val, int memmodel) \
     { \
-        unsigned int mask = irq_disable();    \
-        (void)memmodel;                       \
-        I##n tmp = *(I##n *)ptr;              \
-        *(I##n *)ptr = prefixop(tmp op val);  \
-        irq_restore(mask);                    \
-        return tmp;                           \
+        unsigned int mask = irq_disable();              \
+        (void)memmodel;                                 \
+        I##n tmp = *(volatile I##n *)ptr;               \
+        *(volatile I##n *)ptr = prefixop(tmp op val);   \
```

Comment:
indention of slash

Reply:
Because adding the `volatile` increased the width of the code, the backslashed did no longer align. Therefore I realigned them.

(They are placed on a column which is a multiple of four. This common in the RIOT code base and make the code more visually pleasing. It is, however, not required by the coding convention.)

Reply:
> Therefore I realigned them.

the slash in line 143 is one off

Reply:
> They are placed on a column which is a multiple of four.

My bad: A multiple of four plus one. If the line would be empty except for the backslash, the number of spaces would be a multiple of four.

Reply:
We are talking about the backslashes at the end of the lines, right? I'm not sure if I understand the problem

Reply:
yes... they're not aligned the same way. multiple of four is the right choice ;)
Too bad this is not dealt with by a tool...

Reply:
Vim does deal with this automatically. But it makes spaces to be a multiple of four, so that text after spaces is aligned on a multiple of four + 1. I think this is the consistent way to do it, as the indent of non-backslash characters are also aligned on multiples of four + 1.